docs/how-to-build.md

```diff
@@ -5,6 +5,10 @@ license: "Copyright 2023 Datastrato Pvt Ltd.
 This software is licensed under the Apache License version 2."
 ---
 
+- [Prerequisites](#prerequisites)
```

Comment:
@shaofengshi can you please help to review? Since you have the experience running on Windows.

Reply:
The document looks good for me; I don't know how the performance is on Windows, at least it can help user to compile and run Gravitino.